src/auth/auth.ts

```diff
@@ -200,6 +200,36 @@ export class BaseAuth<T extends AbstractAuthRequestHandler> {
       });
   }
 
+  /**
+   * Gets the user data for the user corresponding to a given provider id.
+   *
+   * See [Retrieve user data](/docs/auth/admin/manage-users#retrieve_user_data)
+   * for code samples and detailed documentation.
+   *
+   * @param providerId The provider ID, for example, "google.com" for the
+   *   Google provider.
+   * @param providerUid The user identifier for the given provider.
+   *
+   * @return A promise fulfilled with the user data corresponding to the
+   *   given provider id.
+   */
+  public getUserByProviderUid(providerId: string, providerUid: string): Promise<UserRecord> {
```

Comment:
Here too. Perhaps `provider` and `providerUid`?

Reply:
This is what was agreed to in the api review, so I'm reluctant to change that. But if we were to do so:

`providerId` has prior art (UserINfo, AuthProviderConfig, getProviderConfig, deleteProviderConfig, updateProviderConfig) so I'd prefer to leave that alone.

`providerUid` however does not. Although a bit overloaded, we could use simply `uid` here instead. I think it's sufficiently clear.

If we do want to change this, we should also revisit the bulk get PR which uses similar names.

wdyt?

Reply:
I'm not a fan of uid, as it may be confused with Firebase uid (see getUser(uid: string), for example). Having a different name for provider uid would make the distinction explicit.